server/middleware/authenticate.js

```diff
@@ -2,16 +2,12 @@ let User = require('./../DataBase/models/user');
 let Token = require('./../DataBase/models/tokens');
 const jwt = require('jsonwebtoken');
 const {statusCodes, messages, secretKeys} = require ('../utilities/constants');
-const {isEmpty} = require('./../utilities/utilityFunctions');
+const {isEmpty, decodeToken} = require('./../utilities/utilityFunctions');
 
 const findByToken = function(token,id,req) {
     let decoded = {};
```

Comment:
remove this let and declatre where you are getting the value

Reply:
Done